ReleaseTooling/Sources/ZipBuilder/ModuleMapBuilder.swift

```diff
@@ -49,8 +49,8 @@ struct ModuleMapBuilder {
 
       if module == "FirebaseFirestoreInternal" {
         content += """
-          link framework "BoringSSL-GRPC"
-          link framework "gRPC-Core"
+          link framework "openssl_grpc"
+          link framework "grpc"
           link framework "grpcpp"
```

Comment:
Should this be grpcc?

Reply:
Nope - https://github.com/grpc/grpc/blob/master/gRPC-C%2B%2B.podspec#L44

Reply:
gRPC-Core -> grpc
gRPC-C++ -> grpcpp

Reply:
So I believe this is ok as is.

Reply:
The changelog is wrong though. Will fix it to `grpcpp` in another PR.

Reply:
I knew I had seen grpcc somewhere today. 😄